pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -700,6 +701,12 @@ class SubDatetime(datetime):
         expected = Timestamp(2000, 1, 1)
         assert result == expected
 
+    @pytest.mark.skipif(not compat.PY38, reason="Requires python 3.8 and above")
+    def test_constructor_fromisocalendar(self):
+        result = Timestamp.fromisocalendar(2000, 1, 1)
+        expected = Timestamp("2000-01-03 00:00:00")
+        assert result == expected
```

Comment:
this test is good.  also pls check that the implementation matches the datetime implementation, something like

```
ex_stdlib = datetime.fromisocalendar(2000, 1, 1)
assert result == ex_stdlib
```

Reply:
can you add a comment at the top of this test "# GH#30395" to point back to this thread